File: UVa/11172-RelationalOperator/p11172.py
cant_values = input()

while cant_values:
    values = raw_input()
    values = values.split(' ')
    first = int(values[0])
    last = int(values[1])

    if first < last:
        print ('<')
    elif first > last:
        print ('>')
    else:
        print ('=')

    cant_values -= 1

File: UVa/11172-RelationalOperator/p11172.java
import java.util.*;
import java.lang.*;

class Main{

    public static void main(String[] args) throws java.lang.Exception {
        Scanner sc = new Scanner(System.in);
	long cantidad = sc.nextInt();
	while(cantidad > 0){
	    long first = sc.nextInt();
	    long last = sc.nextInt();
	    if(first < last) System.out.println("<");
	    else if(first > last) System.out.println(">");
	    else System.out.println("=");
	    --cantidad;
        }
	sc.close();
    }
}
